pages/common/supervisorctl.md

```diff
@@ -4,6 +4,10 @@
 > Supervisorctl is the command-line client piece of the supervisor which provides a shell-like interface.
 > More information: <http://supervisord.org>.
 
+- Get all process status info:
```

Comment:
```suggestion
- Get status information of all processes:
```
Would this be better?

Reply:
Thanks for your comment! I am okay with either one. FYI, `Get all process status info` is from the origin help message of the command.
```
supervisorctl help status
status <name>           Get status for a single process
status <gname>:*        Get status for all processes in a group
status <name> <name>    Get status for multiple named processes
status                  Get all process status info
```

Reply:
I think we can go with the latter even if it's different from the origin help page, as it would better fit with other descriptions.